hack/verify-jsonformat.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+cd "${OS_ROOT}"
+
+json_files=$(find {api,examples,docs,images,plugins,test} -name "*.json")
+tmp_dir=$(mktemp -d --suffix=jsonformat)
+found=0
+
+set +e
+
+enable_formatting="${1:-""}"
+fix_formatting="${2:-""}"
+
+for f in $json_files; do
+  tmp_file="${tmp_dir}$(basename $f)"
+  go run ./hack/jsonformat.go ${f} > ${tmp_file}
+  result=$?
+  if [ "${enable_formatting}" == "--with-formatting" ]; then
```

Comment:
nah, if you do not pass --format that --fix won't fix errors for you ;-) (like a missing bracket or ',', etc...)